src/rep.c

```diff
@@ -379,9 +440,4 @@ r_obj* new_unrep_data_frame(r_obj* key, r_obj* times, r_ssize size) {
 
 // -----------------------------------------------------------------------------
 
-void vctrs_init_rep(r_obj* ns) {
-  args_times_ = new_wrapper_arg(NULL, "times");
-}
-
-static
-struct vctrs_arg* const p_args_times = &args_times_;
+void vctrs_init_rep(r_obj* ns) { }
```

Comment:
I'm fine with removing this altogether now. We can add it back if we need it

Reply:
I think it's ok to leave it there, and then it's less work if we need it again. I.e. with ideal tooling every file would get a init hook automatically.